source/adapter.screenshare.js

```diff
@@ -75,9 +74,9 @@
     };
 
   } else if (window.navigator.webkitGetUserMedia) {
-    baseGetUserMedia = window.navigator.getUserMedia;
 
     navigator.getUserMedia = function (constraints, successCb, failureCb) {
+      baseGetUserMedia = window.navigator.getUserMedia;
```

Comment:
@letchoo 
In my case, if I push it out of this function, I get the following error in Chrome 53
``` <<MediaStream>> Failed retrieving stream: TypeError: Illegal invocation```

On a regular gUM, I'm fine with the current version.
For some reason, my local gAR refuses to offer screensharing, so I'm not quite sure about this scenario.